src/main/java/domain/gamer/Dealer.java

```diff
@@ -0,0 +1,28 @@
+package domain.gamer;
+
+public class Dealer extends Gamer {
+
+    public static final int THRESHOLD = 16;
```

Comment:
```suggestion
    private static final int THRESHOLD = 16;
```
위에서 말씀드린 이유로 상수는 테스트에 활용하지는 않습니다. 도메인 코드와 테스트는 서로 정보를 공유하여 사용하지 않아요. 그래야 변경에 대해 테스트 피드백을 받을 수 있으니까요.